### PR TITLE
Char probs outline

### PR DESCRIPTION
**Pull request by munkm**
Hey, this is the order I imagine the characterization problems chapter going in. Let me know if you'd like me to elaborate on more, since I don't have it exactly paragraph by paragraph. If you think section 4.3 would be better as another separate angle-sensitivity chapter, let me know too. 

- I don't have the parametric study problems chosen or defined in this outline yet because I'm still not done going through the char_probs data. I'd like to get this outline to you ASAP though so I just put in some default names.

### CODE REVIEW

File: chapters/characterization_tests/angle_sensitivity.tex [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+[Show flux maps for each extreme-valued problem in region of interest.] \\
+
+[Describe results.] \\
+
+\subsection{Quadrature Order}
+\label{subsec:quadorder}
+[Show FOMs as a function of quad order for first problem] \\
+
+[Show interesting anisotropy plots for extreme-valued quad types.] \\
+
```
Comment (rachelslaybaugh) [inline chapters/characterization_tests/angle_sensitivity.tex:38]: depending on how many plots come from the extreme valued stuff, you might want to put some in appendix

**Comment by rachelslaybaugh** [comment]
I think the outline makes sense. Sometimes things need re-ordered once you see all of the results, but I think this seems clear. 

I don't think the angle stuff _needs_ to be a separate chapter. However, if we can the demo problem, it might make sense to be its own chapter. If there is a lot in the angle parameter investigation to discuss and show, then it might be worth it. 

Let me know if you need more feedback.

File: chapters/characterization_tests/angle_sensitivity.tex (hunk 1 above)
Comment (munkm) [reply chapters/characterization_tests/angle_sensitivity.tex:38]: Good point. My thoughts on this are maybe that there should be a sister appendix to this chapter that includes all of the results from each problem that aren't included in this chapter. How do you feel about that?

**Comment by munkm** [comment]
That feedback is perfect! Thanks for the quick turnaround on this.

**Comment by rachelslaybaugh** [comment]
that makes sense to me. OR--you could host the data on github or figshare
and just include a link to the full set of doi data :)

**Comment by munkm** [comment]
OOOooohhh. THE FUTURE. I.... definitely might do that. Let's see how this chapter goes first....

**Pull request merged**